coupling/services/CouplingCellService.cpph

```diff
@@ -268,15 +268,16 @@ tarch::la::Vector<dim, double> coupling::services::CouplingCellServiceImpl<Linke
   // initialise with position of global first ghost cell
   //    coupling::indexing::CellIndex<3, coupling::indexing::IndexTrait::vector,
   //    coupling::indexing::IndexTrait::local> localIndex({0, 0, 0});
-  I03 localIndex({0, 0, 0});
+/*  I03 localIndex({0, 0, 0});
```

Comment:
Why is this only commented out? (If the code is not needed, it should be removed, IMHO)

Reply:
Done!